tests/acceptance/repo/allowance-test.js

```diff
@@ -14,76 +14,83 @@ module('Acceptance | repo allowance', function (hooks) {
     const currentUser = this.server.create('user', {login: 'user-login'});
     signInUser(currentUser);
 
+    this.server.create('user', {login: 'user-login2'});
+    this.server.create('user', {login: 'user-login3'});
+
     const repoPrivate = this.server.create('repository', {
       name: 'repository-private',
       slug: 'user-login/repository-private',
       'private': true,
       active: true,
       owner: {
         login: 'user-login',
-        allowance: {
-          subscription_type: 2,
-          private_repos: false,
-          public_repos: true,
-          concurrency_limit: 1
-        }
+        id: 1
       },
       vcs_name: 'repository-private',
       owner_name: 'user-login'
     });
 
+    this.server.create('allowance', {
+      id: 1,
+      subscription_type: 2,
+      public_repos: false,
+      private_repos: false,
+      concurrency_limit: 777
+    }),
+
+    this.server.create('allowance', {
+      id: 2,
+      subscription_type: 2,
+      public_repos: true,
+      private_repos: true,
+      concurrency_limit: 666
+    }),
+
+    this.server.create('allowance', {
+      id: 3,
+      subscription_type: 2,
+      public_repos: true,
+      private_repos: false,
+      concurrency_limit: 2
+    }),
```

Comment:
You shouldn't specify `id`s manually, Mirage's factories will generate them automatically and take care of uniqueness.

Reply:
This comment isn't fixed. Although, let's keep it as is for now. It seems to be working with generated ids too, so this is basically just a confusing thing. I'd appreciate if you could clean this up sometime later, don't want to hold this PR because of it.